docs/book/migration/to-3-0.md

```diff
@@ -0,0 +1,22 @@
+# Migration guide
+
+Migration guide for Laminas Cache version 3.0.0.
+
+## New Features
+
+- Each cache adapter has its [own package](#satellite-packages).
+
+## Breaking Changes
+
+- Removal of support for `laminas/laminas-servicemanager` and the related files
```

Comment:
> Removal of support for `laminas/laminas-servicemanager`…

Is this correct or only for the pattern plugin manager?

And if the pattern plugin manager is removed, is there an alternative use, a better solution or was that a bad idea in the past?

Reply:
Oh, I need to add the `v2` postfix.
Only dropped SM v2 support tho.